inc/blokus.h

```diff
@@ -48,6 +63,10 @@ struct _tile_attr {
     int rot_cnt;
 };
 
+struct _status_attr {
+    char *desc;
+};
+
```

Comment:
Oh, no! This should not exist.

Reply:
Or maybe I should add some description strings for each status?

Reply:
It'll be great if you add some descriptions so that I could use them directly at the client side!

Reply:
After thinking a while, I find it more suitable for you to do so, since the game has no knowledge to the rendering and its restrictions.

Reply:
Or say, the code should be written in like `render.h`.